### PR TITLE
Restructuring the AR spec text to improve clarity

### PR DESCRIPTION
**Pull request by NellWaliczek**
While reviewing @rcabanier's PR #19 I was really struck by two things about the current spec text. First, we have a lot of terms that we have been putting in quotes rather than formally defining.  Second, boy oh boy is there a lot of duplicate text.  The intent of this PR is to address both of those things without changing the meaning of any existing spec content.  Which, hopefully, I've done effectively.

Beyond general feedback, there is one thing I'm not super satisfied with and would love to get an opinion on.  I suspect the term on line 131 is too easy to confuse with the actual blend mode enum value.  But every time I try to come up with something else... I just come back to "opaque".  *sigh* Anyone got a suggestion? Also, I'm curious if anyone has concerns that this definition accidentally implies all opaque devices are head-worn.  Am I overthinking it?

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -193,41 +190,32 @@ partial interface XRSession {
...
-- A blend mode of <dfn enum-value for="XREnvironmentBlendMode">alpha-blend</dfn> MUST be reported if the pixels of the {{XRRenderState/baseLayer}} are composited using [=source-over=] during [=perform automatic composition|automatic-composition=].
+- A blend mode of <dfn enum-value for="XREnvironmentBlendMode">additive</dfn> MUST be reported if the [=XR Compositor=] is using [=additive environment blending=].
 
-    NOTE: typically this mode is returned for "pass-through" displays
-
-The <dfn attribute for="XRSession">environmentBlendMode</dfn> attribute returns the {{XRSession}}'s [=environment blending mode=].
-
-Note: Most Virtual Reality devices are only capable of exhibiting {{XREnvironmentBlendMode/"opaque"}} blending behavior. Augmented Reality devices with "additive light" displays frequently exhibit {{XREnvironmentBlendMode/"additive"}} blending behavior even for {{XRSessionMode/"immersive-vr"}} sessions.
-
-ISSUE: Clarify the behavior of inline sessions
```
Comment (rcabanier) [inline index.bs]: did you mean to remove this issue?

File: index.bs [hunk 2]
```diff
@@ -193,41 +190,32 @@ partial interface XRSession {
...
 
-An [=xr device=] during an {{XRSessionMode/"immersive-ar"}} session will have its [=xr compositor=] <dfn>perform automatic composition</dfn> by drawing the rendered scene on top of the [=aligned real-world background|real-world background=].
+When presenting content to the [=XR device=], the [=XR Compositor=] MUST apply the appropriate <dfn>blend technique</dfn> to combine virtual pixels with the [=real-world environment=]. The appropriate technique is determined based on the [=XR device=]'s [=display technology=] and the [=XRSession/mode=].
+
+- When performing <dfn>opaque environment blending</dfn>, the rendered buffers obtained by the [=XR Compositor=] are composited using [=source-over=] blending on top of buffers containing exclusively black pixels. The composited output is then presented on the [=XR device=]. This technique MUST be applied on [=opaque=] and [=pass-through=] displays when the [=XRSession/mode=] is set to either {{XRSessionMode/"immersive-vr"}} or {{XRSessionMode/"inline"}}. This technique MUST NOT be applied when the [=XRSession/mode=] is set to {{XRSessionMode/"immersive-ar"}}, regardless of the [=XR Device=]'s [=display technology=].
```
Comment (rcabanier) [inline index.bs]: black -> 100% opaque black

File: index.bs [hunk 3]
```diff
@@ -193,41 +190,32 @@ partial interface XRSession {
...
+The [=XR Compositor=] MUST NOT automatically grant the page access to any additional information such as camera intrinsics, media streams, real-world geometry, etc.
 
-During automatic composition the [=xr device=] MUST have the capability of producing an [=aligned real-world background=] for each [=view=].
-An [=xr device=] MUST capture the <dfn>aligned real-world background</dfn> in such a way that it can be aligned with the pixels of the {{XRRenderState/baseLayer}}.
+NOTE: Developers may request access to an [=XR Device=]'s camera, should one be exposed through the existing WebRTC specification. However, doing so does not provide a mechanism to query the {{XRRigidTransform}} between the camera's location and the [=native origin=] of the [=viewer reference space=]. It also does not provide a guaranteed way to determine the camera intrinsics necessary to match the view of the [=real-world environment=]. As such, performing effective computer vision algorithms wil be significantly hampered. Future modules or specifications may enable such functionality.
```
Comment (rcabanier) [inline index.bs]: [getUserMedia](https://w3c.github.io/mediacapture-main/#dom-mediadevices-getusermedia) is how a page gets access to the camera, not WebRTC

File: index.bs [hunk 4]
```diff
@@ -119,20 +123,15 @@ Hardware that enables Virtual Reality (VR) and Augmented Reality (AR) applicatio
...
- * Fixed displays with head tracking capabilities
-
-The important commonality between them being that they offer some degree of spatial tracking with which to simulate a view of virtual content.
-
-Terms like "XR Device", "XR Application", etc. are generally understood to apply to any of the above. Portions of this document that only apply to a subset of these devices will indicate so as appropriate.
+XR hardware can be divided into categories based on <dfn>display technology</dfn>.
+- Devices described as <dfn>additive light</dfn>, also known as see-through, use transparent optical displays to present virtual content. On these devices, the user may always be able to see through to the [=real-world environment=] regardless of developer requests during session creation.
+- Devices described as <dfn>pass-through</dfn> use an opaque display to combine virtual content with a camera stream of the [=real-world environment=]. On these devices, the [=real-world environment=] will only be visible when the developer has made an explicit request for it during session creation.
+- Devices described as <dfn>opaque</dfn> which obscures the [=real-world environment=].
```
Comment (rcabanier) [inline index.bs]: not: obscures -> obscure

**Comment by rcabanier** [comment]
> I just come back to "opaque". _sigh_ Anyone got a suggestion? 

I can't think of a better term either :-\
This section is not normative, right? Should we mark it as such?

> Also, I'm curious if anyone has concerns that this definition accidentally implies all opaque devices are head-worn. Am I overthinking it?

Since you don't explicitly call out that any device type is hand-held or head-worn, I don't think this is implied.

**Comment by NellWaliczek** [comment]
Thanks for your feedback @rcabanier!  If this looks good, I'll get it merged today.

**Comment by rcabanier** [comment]
> Thanks for your feedback @rcabanier! If this looks good, I'll get it merged today.

Looks good!

**Pull request merged**